### ISSUE TITLE
ctor.dump of Union is not working correctly

### ISSUE THREAD

**Issue by mrmerkone** [bshishov/ctor#1]
``` python

import ctor
from dataclasses import dataclass

@dataclass
class A:
    a: str


@dataclass
class B:
    b: int


@dataclass
class Foo:
    item: Union[A, B]


foo = Foo(item=B(1))


print(ctor.dump(foo))

# expected: {'item': {'b': 1}}
# actual: {'item': {}}
```

**Comment by bshishov** [comment]
Fixed in v0.3.1

**Comment by mrmerkone** [comment]
Pogchamp
